src/components/List.js

```diff
@@ -1,5 +1,83 @@
 import React from 'react'
+import { connect } from 'react-redux'
+import Task from './Task/Task'
+import AppPoup from './Popup'
+import MainButton from './Buttons/MainButton'
+import { removeTask, addTask } from '../redux/reducers/todos/actions'
 
-const List = () => <div>List</div>
+const mapStateToProps = ({ todos }) => {
+  return { tasks: todos.tasks }
+}
 
-export default List
+const mapDispatchToProps = dispatch => ({
+  remove: id => dispatch(removeTask(id)),
```

Comment:
You can assign object to mapDispatchToProps and it will dispatch methods automatically.
```
const mapDistaptchToProps = {
  remove: removeTask,
  add: addTask
}
```